site/pages/api/delete-disruption.api.ts

```diff
@@ -1,19 +1,33 @@
 import { PublishStatus } from "@create-disruptions-data/shared-ts/enums";
 import { NextApiRequest, NextApiResponse } from "next";
-import { DASHBOARD_PAGE_PATH, ERROR_PATH, VIEW_ALL_TEMPLATES_PAGE_PATH } from "../../constants";
+import {
+    DASHBOARD_PAGE_PATH,
+    DISRUPTION_DETAIL_PAGE_PATH,
+    ERROR_PATH,
+    VIEW_ALL_TEMPLATES_PAGE_PATH,
+} from "../../constants";
 import { deletePublishedDisruption, getDisruptionById } from "../../data/dynamo";
-import { isDisruptionFromTemplate, redirectTo, redirectToError } from "../../utils/apiUtils";
+import { redirectTo, redirectToError } from "../../utils/apiUtils";
 import { canPublish, getSession } from "../../utils/apiUtils/auth";
 import logger from "../../utils/logger";
 
+const getRedirectPath = (template: string, returnPath: string) => {
+    if (template) {
+        return VIEW_ALL_TEMPLATES_PAGE_PATH;
+    }
+    if (returnPath?.toString().includes(DISRUPTION_DETAIL_PAGE_PATH) && returnPath?.toString().includes("template")) {
```

Comment:
Would a truthy check be sufficient to redirect to the path mentioned in return param?

Reply:
I didn't want to do this because if you click an existing disruption, then click a change link to edit the disruption a returnPath will be present in the URL that will redirect you have to the disruption overview page. 

So if I change this to a truthy check we'll get thrown a 404 as it will try to take us to a disruption overview page for a disruption we deleted.